add_npm_package.sh

```diff
@@ -91,6 +90,11 @@ add_npm_to_comps() {
   git add comps/
 }
 
+npm_info() {
+  local name=$(python -c "import urllib ; print(urllib.quote('$NPM_MODULE_NAME', safe=''))")
```

Comment:
this depends on an external variable `NPM_MODULE_NAME` being set, any reason not to use `$1` and then calling `npm_info $NPM_MODULE_NAME`?

Reply:
Good point.